src/components/Onboarding/OnboardingChecklist/index.tsx

```diff
@@ -0,0 +1,158 @@
+import { useEffect } from 'react';
+
+import classNames from 'classnames';
+import useTranslation from 'next-translate/useTranslation';
+import { useRouter } from 'next/router';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { useOnboarding } from '../OnboardingProvider';
+import { onboardingChecklist } from '../steps';
+
+import styles from './OnboardingChecklist.module.scss';
+
+import Button, { ButtonShape, ButtonSize, ButtonVariant } from '@/dls/Button/Button';
+import CheckIcon from '@/icons/check.svg';
+import IconClose from '@/icons/close.svg';
+import IconQuestionMark from '@/icons/question-mark-icon.svg';
+import {
+  dismissChecklist,
+  selectOnboarding,
+  setIsChecklistVisible,
+} from '@/redux/slices/onboarding';
+import OnboardingGroup from '@/types/OnboardingGroup';
+import { logButtonClick, logEvent } from '@/utils/eventLogger';
+
+const OnboardingChecklist = () => {
+  const { t } = useTranslation('common');
+  const checklistItems = onboardingChecklist(t);
+  const router = useRouter();
+
+  const dispatch = useDispatch();
+  const { allSteps, startTour, isActive } = useOnboarding();
+  const {
+    isChecklistVisible,
+    checklistDismissals,
+    lastChecklistDismissal,
+    groups: completedGroups,
+  } = useSelector(selectOnboarding);
+
+  const handleDismiss = () => {
+    logButtonClick('onboarding_checklist_dismiss', { totalDismissals: checklistDismissals + 1 });
+    dispatch(dismissChecklist({ countDismiss: true }));
+  };
+
+  const openChecklist = () => {
+    logButtonClick('onboarding_checklist_open');
+    dispatch(setIsChecklistVisible(true));
+  };
+
+  useEffect(() => {
+    // If the user didn’t finish onboarding then we should show the onboarding again after 1 month from the last time. Repeat this behaviuor for 3 months (once per month) and finally, the system should not alert the user anymore for this onboarding.
+    const didNotFinishOnboarding = !!checklistItems.find(({ group }) => {
+      const completedSteps = completedGroups[group]?.completedSteps;
+      return completedSteps?.length !== allSteps[group].length;
+    });
+
+    if (
+      checklistDismissals >= 3 ||
+      isChecklistVisible ||
+      !lastChecklistDismissal ||
+      !didNotFinishOnboarding
+    ) {
+      return;
+    }
+
+    const now = Date.now();
+    const oneMonth = 1000 * 60 * 60 * 24 * 30;
```

Comment:
nit: move `oneMonth` into a const outside the `useEffect` and rename it to `showCheckListAfter` or something similar in-case `oneMonth` changes.